src/promptflow-core/promptflow/executor/_prompty_executor.py

```diff
@@ -20,7 +20,7 @@ class PromptyExecutor(ScriptExecutor):
 
     def __init__(
         self,
-        flow_file: Path,
+        flow_file: Union[Path, Callable],
```

Comment:
maybe rename this param, flow_file is confusing to accept Callable

Reply:
This is inherited from the naming in ScriptExecutor

Reply:
maybe change in future